packages/react/src/SkipLink/SkipLink.test.tsx

```diff
@@ -0,0 +1,42 @@
+import { render, screen } from '@testing-library/react'
+import { createRef } from 'react'
+import { SkipLink } from './SkipLink'
+import '@testing-library/jest-dom'
+
+describe('Skip link', () => {
+  it('renders', () => {
+    render(<SkipLink href="/" />)
+
+    const component = screen.getByRole('link')
+
+    expect(component).toBeInTheDocument()
+    expect(component).toBeVisible()
+  })
+
+  it('renders a design system BEM class name', () => {
+    render(<SkipLink href="/" />)
+
+    const component = screen.getByRole('link')
+
+    expect(component).toHaveClass('amsterdam-skip-link')
+  })
+
+  it('renders an additional class name', () => {
+    render(<SkipLink href="/" className="extra" />)
+
+    const component = screen.getByRole('link')
+
+    expect(component).toHaveClass('extra')
+    expect(component).toHaveClass('amsterdam-skip-link')
```

Comment:
I think this is equivalent.

```suggestion
    expect(component).toHaveClass('amsterdam-skip-link extra')
```

Reply:
Yeah, good idea. I changed it in the plop template as well